app/src/main/java/by/academy/jc/akbarova/prime_number/Prime.java

```diff
@@ -0,0 +1,20 @@
+package by.academy.jc.akbarova.prime_number;
+
+public class Prime {
+    public static int isPrime(int a, int b) {
```

Comment:
Use some of the available prime number algorithms like https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes